### PR TITLE
commit form

### PR DESCRIPTION
**Pull request by daryna-kopytko**
https://daryna-kopytko.github.io/layout_html-form/
https://daryna-kopytko.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,6 +12,182 @@
...
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </div>
+
+      </fieldset>
+
+      <input type="submit"
+        name="submit"
+      >
```
Comment (ivanchuk-yurii) [inline src/index.html:188]:
```suggestion
      <input
        type="submit"
        name="submit"
      >
``` 
or all in one line

File: src/style.css [hunk 2]
```diff
@@ -1 +1,8 @@
-/* styles go here */
+
+.inputs {
+  margin: 10px;
+}
+
+fieldset {
```
Comment (ivanchuk-yurii) [inline src/style.css:6]: style classes, not tags

File: src/index.html [hunk 3]
```diff
@@ -12,6 +12,182 @@
...
+            rows="3"
+          >
+          </textarea>
+        </div>
+
+        <div>
+          Would you recommend us?
+          <select
+          class="inputs"
+          name="recommendation">
```
Comment (ivanchuk-yurii) [inline src/index.html:178]:
```suggestion
          <select
            class="inputs"
            name="recommendation"
          >
```

File: src/index.html [hunk 4]
```diff
@@ -12,6 +12,182 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
```
Comment (ivanchuk-yurii) [inline src/index.html:19]: use labels

File: src/index.html [hunk 5]
```diff
@@ -12,6 +12,182 @@
...
+        </div>
+
+        <div>
+          I accept the term of the agreement
+          <input
+            class="inputs"
+            type="checkbox"
+          >
+        </div>
+
```
Comment (ivanchuk-yurii) [inline src/index.html:67]: no need in empty line here

File: src/style.css [hunk 6]
```diff
@@ -1 +1,8 @@
-/* styles go here */
+
```
Comment (ivanchuk-yurii) [inline src/style.css:1]: no empty line here

File: src/index.html (hunk 1 above)
Comment (ivanchuk-yurii) [reply src/index.html:188]: use button with text 'Submit'

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
fix tests and compare your layout with task description screenshot

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Nothing has changed since the last review.
If you have questions, ask in the fe_chat

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
I don't see any commits. Did you do them?
If you have questions, please ask in fe_chat

**Pull request closed without merging**